src/Content/Text/BBCode.php

```diff
@@ -111,7 +111,7 @@ private static function getOldAttachmentData($body)
 
 					$picturedata = Images::getInfoFromURLCached($matches[1]);
 
-					if ($picturedata) {
+					if (!empty($picturedata)) {
```

Comment:
Empty array gets casted as false in a condition, what is the purpose of `empty()` here?

Reply:
I normally don't like this "magic casting" thing and try to make it clean (with no casting). Plus also in other occurrences I found the `empty($array)` calls. Should I then rewrite all to one way?

Reply:
I would like to make each handling of returned variables from `Imagess::getInfoFromURL[Cached]()` the same way, that we know this is everywhere the same and don't have to check each case.